strongbox-distribution/src/main/assembly/distribution.xml

```diff
@@ -62,14 +47,6 @@
             <fileMode>0755</fileMode>
             <lineEnding>dos</lineEnding>
         </fileSet>
-        <fileSet>
-            <directory>target/assembly/strongbox-${project.version}/bin</directory>
-            <outputDirectory>strongbox-${project.version}/bin</outputDirectory>
```

Comment:
@sbespalov , @steve-todorov ,

Just a reminder-question: Where do things stand with the service scripts? Do we have all the necessary follow-up tasks? Could you please link back (here) to the issues where this was done? Thanks! :)

Reply:
#1459

Reply:
Thanks! :)